pom.xml

```diff
@@ -17,8 +17,8 @@
         <url>https://github.com/vackosar/gitflow-incremental-builder</url>
         <connection>scm:git:https://github.com/vackosar/gitflow-incremental-builder.git</connection>
         <developerConnection>scm:git:https://github.com/vackosar/gitflow-incremental-builder.git</developerConnection>
-      <tag>HEAD</tag>
-  </scm>
+        <tag>HEAD</tag>

```

Comment:
The next release will "break" this again. The release plugin is reformatting this file.

Reply:
I'll revert this change.